style.css

```
@@ -0,0 +1,185 @@
+#header .wrap-header {
```

💊 제안
아이디 선택자는 우선순위가 높아 덮어씌우기 쉽지 않아 사용을 지양하는 것이 좋습니다.
가능하면 classname을 사용해주세요~

```suggestion
.wrap-header {
```